requirements/base.txt

```diff
@@ -0,0 +1,2 @@
+aiohttp<2.3
```

Comment:
Why `<2.3`?
Does aiodocker work on aiohttp 1.0? Is it broken on aiohttp master?